packages/cli/src/tests/utils/analytics.js

```diff
@@ -0,0 +1,9 @@
+require('should');
+const { shouldSkipAnalytics } = require('../../utils/analytics');
+
+describe('analytics', () => {
+  // causes a lot of noise
+  it('should not run analytics when testing', () => {
+    shouldSkipAnalytics().should.be.true();
```

Comment:
this makes the functionality tough to test. it fixes the problem for now, but we'll probably have to rethink this approach.